desktop/containerd/index.md

```diff
@@ -209,6 +209,19 @@ View Tags on DockerHub to see multi-platform result:
 
 ## Containerd Image Store Release Notes
 
+(2022-10-19)  
+New commands and multiple bug fixes included with Docker Desktop 4.13.0.
+
+### New
+* Fixed: Kubernetes not starting.
```

Comment:
@bsousaa Perhaps it's worth creating a new section called 'Bug fixes' and moving most of these bullet points?

Reply:
Thanks @aevesdocker ! Done.